python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -29,8 +30,9 @@ class _PipesSubprocess(PipesClient):
     a temp file, and structured messages are read from from a temp file.
 
     Args:
-        env (Optional[Mapping[str, str]]): An optional dict of environment variables to pass to the
-            subprocess.
+        env (Optional[Mapping[str, str]]): The base environment variables to pass to the
```

Comment:
maybe should rename to `base_env` if we go with this approach

Reply:
One way to think about naming is what is the scenario where we imagine the `__init__` `env` arg, a callsite-specific arg, and inheriting `os.environ` at the callsite interacting. Why use this one versus pushing everything to the callsite?